.github/workflows/pypi-publish.yml

```
@@ -0,0 +1,35 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  Publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel
+
+      - name: Build package
+        # Remove `compile_catalog` if the package has no translations.
+        run: |
+          python setup.py compile_catalog sdist bdist_wheel
```

The module doesn't have translations:
```suggestion
          python setup.py sdist bdist_wheel
```

Perhaps you can rerun the ga-migration script as this is fixed there.

---

@diegodelemos Done